rocketbelt/components/images/_images-lazyload.scss

```diff
@@ -46,7 +46,15 @@
 
     &.lazyload-small {
       opacity: 0.5;
-      transition: opacity 200ms ease-in-out;
+      transition: opacity 200ms ease-in-out;     
+    }
+    
+    &.errored{
+      background: none;
+      animation: none;
+      
+      & img.lazyload-small{
+        opacity: 1;
+      }
     }
-  }
```

Comment:
This brace needs to stay for the file to compile!

Reply:
must have missed this when i copied my changes over from notepad++